client/layout/masterbar/logged-in.jsx

```diff
@@ -1,6 +1,7 @@
 /**
  * External dependencies
  */
+import qs from 'qs';
```

Comment:
We might want to consider only importing the method we need: `import { parse } from 'qs';`.

Reply:
Good point, fixed!